src/platform/user/tests/profile/actions/index.unit.spec.js

```diff
@@ -0,0 +1,117 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import * as actions from 'platform/user/profile/actions'; // Replace with actual path
```

Comment:
does the path from the linter warnings work for you?  I ran into some trouble with this before, but i think that was more related to the mjs vs cjs modules issue. just wondering if you've tried to use the `@department-of-veterans-affairs/platform-user` alias yet?

Reply:
honestly I have been ignoring those linting warnings. 

I tried to adopt them when they were first rolled out, and they had issues and didn't allow me autocomplete for exports, follow the import to the source via  ctrl+click, and other issues.